packages.yml

```
@@ -1,3 +1,3 @@
 packages:
   - package: fivetran/pinterest_source
-    version: [">=0.7.0", "<0.8.0"]
\ No newline at end of file
+    version: [">=0.8.0", "<0.9.0"]
```

I believe this should be 9 and 10
```suggestion
    version: [">=0.9.0", "<0.10.0"]
```